ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_internet_exit.schema.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  router_internet_exit:
+    type: dict
+    description: Internet-exit feature provides internet bound service offered in an AVT to the traffic going outside the enterprise towards the internet.
+    keys:
+      policies:
+        type: list
+        description: Internet-exit policy represent a policy which can be attached to an AVT profile that needs to be applied to the traffic matching the AVT.
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+            exit_groups:
+              type: list
+              description: The exit groups that are configured under a policy are strictly ordered, meaning an exit group appearing first has more priority than the exit group that follows it.
+              primary_key: name
+              items:
+                type: dict
+                keys:
+                  name:
+                    type: str
+                  after:
+                    type: str
+                    description: Insert the previous exit group AFTER the following exit group.
+                  before:
+                    type: str
+                    description: Insert the previous exit group BEFORE the following exit group.
```

Comment:
We don't do this in AVD - people need to declare the groups in the right order.
Keep the structure a dict

Reply:
Will remove 
```
                  after:
                    type: str
                    description: Insert the previous exit group AFTER the following exit group.
                  before:
                    type: str
                    description: Insert the previous exit group BEFORE the following exit group.
       
```

but need to keep the **exit_groups** structure a list  since multiple entries of **exit_groups** is supported in a policy.

Reply:
yes list of dict as you have today :)

Reply:
Done!